A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.css

```diff
@@ -97,6 +97,22 @@ a320-neo-pfd-element {
         height: 48%;
         border: none;
         transform: scale(0.9); }
+      a320-neo-pfd-element #Mainframe #HorizonContainer #GroundCursor {
+        position: absolute;
+        left: 40.9%;
+        top: 47.7%;
+        width: 7%;
+        height: 7%;
+        border: none;
+        display: block; }
```

Comment:
Please put the curly brace on the next line, the instances of it being on the same line is from original game code :)

Reply:
its fine to stay consistent with the existing code